nixos/modules/services/desktops/gnome/gnome-keyring.nix

```diff
@@ -36,6 +36,7 @@ in
     xdg.portal.extraPortals = with pkgs; [ gnome.gnome-keyring ];
 
     security.pam.services.login.enableGnomeKeyring = true;
+    security.pam.services.gdm-password.enableGnomeKeyring = config.services.xserver.displayManager.gdm.enable;
```

Comment:
Arch adds this as well so it is probably fine https://gitlab.gnome.org/GNOME/gdm/-/blob/main/data/pam-arch/gdm-password.pam#L4

Reply:
Nice. But I believe this should only be enabled if GDM is used?

Some users (without GNOME) may prefer another login manager such as SDDM, in which case `login` PAM should work.

Reply:
I enabled gnome-keyring for `autologin`, `fingerprint` and `smartcard` authentication mode too.

Reply:
Right, I just meant that it should be okay to include keyring in `gdm-password`. Will need to double check the other auth modes, in case their presence signals specific behavour to GDM (we previously did not have those files).

Reply:
I can't test fingerprint and smartcard, but autologin seems fine.